torch/_inductor/codegen/common.py

```diff
@@ -99,6 +99,15 @@ def index_prevent_reordering(index: List[sympy.Expr], index_vars, sizes):
     return [*index, sympy_dot(index_vars, FlexibleLayout.contiguous_strides(sizes))]
 
 
+def get_device_op_overrides(device: str):
+    if device == "cuda":
```

Comment:
```suggestion
    assert isinstance(device, str)
    if device == "cuda":
```

to make sure people don't call this with torch.device by mistake